Documentation/ApiOverview/Logging/Writers/Index.rst

```diff
@@ -172,25 +172,50 @@ the :ref:`PhpErrorLog <logging-writers-php>` writer).
 Usage in Custom Class
 ---------------------
 
-All log writers can be used in your own classes. You can initialize the loggers in the __construct method of your class :code:`\MyDomain\MyExtension\MyFolder\MyClass`.
+All log writers can be used in your own classes. If the service is configured to use autowiring 
+you can inject a logger into the :php:`__construct` method of your class :php:`\MyVendor\MyExtension\MyFolder\MyClass`) since TYPO3 v11 LTS.
+
 
 .. code-block:: php
 
     namespace MyDomain\MyExtension\MyFolder;
 
     use Psr\Log\LoggerInterface;
 
-    class MyClass implements {
+    class MyClass {
        private LoggerInterface $logger;
 
        public function __construct(LoggerInterface $logger) {
            $this->logger = $logger;
        }
         ...
-        $this->logger->info('My class is executed.');
-        if ($error) {
-           $this->logger->error('error in class MyClass');
-        }
+           $this->logger->info('My class is executed.');
+           if ($error) {
+              $this->logger->error('error in class MyClass');
+           }
+        ...
+    }
+
+
+If autowiring is disabled, the service class however must implement the interface :php:`\Psr\Log\LoggerAwareInterface` and use the :php:`\Psr\Log\LoggerAwareTrait`.
+
+
+.. code-block:: php
+
+    namespace MyVendor\MyExtension\MyFolder;
+
+    use Psr\Log\LoggerAwareInterface;
+    use Psr\Log\LoggerAwareTrait;
+
+    class MyClass implements LoggerAwareInterface {
+       use LoggerAwareTrait;
+
+        ...
```

Comment:
Here a method is now missing

Reply:
Yup, it was before also. Will do a follow-up to clean also other things up.

Reply:
Which method is missing?

Reply:
We will do that in a follow up, @brotkrueml can tag you in the follow up